.goreleaser.yaml

```diff
@@ -25,6 +25,8 @@ builds:
       - linux
       - windows
       - darwin
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.ShortCommit}} -X main.builtBy=goreleaser'
```

Comment:
It is imported from goreleaser default https://github.com/goreleaser/goreleaser/blob/b58a766d83563b29b58c0b9bacb841cabde516ec/www/docs/customization/builds.md?plain=1#L55

But I agree we can drop it, because of I already omitted date field 🙇‍♂️